snapcraft/cli/store.py

```diff
@@ -252,18 +278,91 @@ def list_registered():
     snapcraft.list_registered()
 
 
+@storecli.command('export-login')
+@click.argument('login_file', metavar='FILE', type=click.File('w'))
+@click.option('--packages', metavar='<packages>',
+              help='Comma-separated list of packages to limit token access')
+@click.option('--channels', metavar='<channels>',
+              help='Comma-separated list of channels to limit token access')
+@click.option('--acls', metavar='<acls>',
+              help='Comma-separated list of ACLs to limit token access. Valid '
+                   "ACLs are 'package_upload', 'package_access', and "
+                   "'package_manage'")
+def export_login(login_file: TextIO, packages: str, channels: str, acls: str):
+    """Save attenuated login configuration for a store account in FILE.
+
+    This file can then be used to login to the given account with the
+    permissions specified.
+
+    For example, to limit access to the edge channel of any snap to which the
+    account has access:
```

Comment:
Maybe “of any snap that the account has access to:”

Reply:
I generally try to avoid ending with a preposition, although I agree that this is too wordy. Perhaps "of any snap the account can access" ?